Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoIse.yaml

```diff
@@ -0,0 +1,84 @@
+    let ASIM_GetUsernameType = (username:string) { 
+      case ( 
+          username contains "@" , "UPN"
+          , username contains "\\", "Windows"
+          , isempty(username), ""
+          , "Simple"
+      )
+    };
+    let ASIM_GetSrcHostname = (message:string) {
+      case(
+          message matches regex @"Caller-Station-ID=\d{1,3}\.\d{1.3}\.\d{1,3}\.\d{1,3}", "",
+          message matches regex @"Caller-Station-ID=[a-fA-F0-9\-:]{17}", "",
+          tostring(extract(@"Caller-Station-ID=(.*?)[:,]", 1, message))
+      )
+    };
+    let CiscoISEAuthParser=(disabled:bool=false){
+      Syslog
+      | where not(disabled)
+      | where ProcessName has_any ("CISE", "CSCO")
+      | extend EventOriginalType = extract(@".*\d{10}\s(\d{4,5})\s.*", 1, SyslogMessage)
```

Comment:
We should use parse statement for EventOriginalType and EventOriginalSeverity. Remaining string can be used for parse-kv.

Reply:
`EventOriginalSeverity` moved to lookup table. parse is used for `EventOriginalType`

Reply:
For the remaining fields  `parse-kv` is used